### PR TITLE
doc_models_1.0

### PR DESCRIPTION
**Pull request by antotaverna**

### CODE REVIEW

File: galaxychop/models.py [hunk 1]
```diff
@@ -114,7 +124,30 @@ class GCClusterMixin(GCDecomposeMixin, ClusterMixin):
...
+
+    Implementation of galaxy dynamical decomposition model described in
+    Abadi et al. (2003) [1]_.
+
+    Examples
+    --------
+    Example of implementation of Abadi Model.
+
+    >>> gal0 = gc.Galaxy(...)
+    >>> gcabadi = gc.GCAbadi()
```
Comment (vcristiani) [inline galaxychop/models.py:137]: gcabadi = gc.GCAbadi(n_bin=100, digits=2, seed=None)

File: galaxychop/models.py [hunk 2]
```diff
@@ -290,14 +323,54 @@ def transform(self, X, y=None):
...
         Cut-off value in the circularity parameter. Particles with
         eps>eps_cut are assigned to the disk and particles with eps<=eps_cut
         to the spheroid.
+
+    Examples
+    --------
+    Example of implementation of Chop Model.
+
+    >>> gal0 = gc.Galaxy(...)
+    >>> gcchop = gc.GCChop()
```
Comment (vcristiani) [inline galaxychop/models.py:345]: gcchop = gc.GCChop(eps_cut=0.6)

File: galaxychop/models.py [hunk 3]
```diff
@@ -366,7 +493,32 @@ def fit_transform(self, X, y=None):
...
+
+    >>> gal0 = gc.Galaxy(...)
+    >>> gcautogmm = gc.GCAutogmm()
+    >>> X, y = gal0.values()
+    >>> preguntar_vale
+    >>> gcautogmm.fit(X)
+    >>> gcautogmm.transform()
+    >>> labels = gcautogmm.labels_
+    >>> probability = gcautogmm.probability
+    >>> preguntar_vale
```
Comment (vcristiani) [inline galaxychop/models.py:513]: >>> gal0 = gc.Galaxy(...)
    >>> gcautogmm = gc.GCAutogmm(c_bic=0.1)
    >>> gcautogmm.decompose(gal0)
    >>> labels = gcautogmm.labels_
    >>> print(labels)
    array([-1, -1,  2, ...,  1,  2,  1])

**Review by vcristiani — COMMENTED** [review]
I added some comments that I think to make it clearer how each decomposer works.
